vip_spec.xsd

```diff
@@ -200,6 +202,7 @@ reference elements for polling location
             <xs:sequence>
               <xs:element name="PartyID" type="xs:IDREF" minOccurs="0" />
               <xs:element name="PersonID" type="xs:IDREF" minOccurs="0" />
+              <!-- SURVEY: Should external identifiers be referenced or included inline? -->
```

Comment:
Okay. I think that should be a different PR, but I just wanted to get everyone's opinion here.